pyServer/GuestFS.py

```diff
@@ -37,7 +37,7 @@ def buildGFArgs(self, args):
         retArgs.extend(args)
         return retArgs
 
-    def callGF(self, echoStr, commands, continueOnError=False, returnRawResults=False):
+    def callGF(self, echoStr, commands, continueOnError=False, returnRawResults=False, retryCount=0):
```

Comment:
Interesting.  I was expecting it to increment to some defined maximum.  As you say, this allows retries to be configurable by individual GF call, which might be useful.  Perhaps we can add a comment or make the variable naming more reflective?  Something like `allowableRetryCount` or `retriesRemainingCount`?

Reply:
How about this as variable name, reversing one of your suggested - "remainingRetriesCount" ??